src/database/queries/ChapterQueries.ts

```diff
@@ -348,7 +349,7 @@ export const deleteDownloads = async (chapters: DownloadedChapter[]) => {
   );
   db.transaction(tx => {
     tx.executeSql('UPDATE Chapter SET isDownloaded = 0', [], () =>
-      showToast('Deleted all Downloads'),
+      showToast(getString('toast.deletedAllDownloads')),
```

Comment:
It would be inconsistent if you dont use string `toast.*` for the other showToasts